.github/renovate.json

```
@@ -1,22 +1,3 @@
 {
-  "extends": [
-    "config:base"
-  ],
-  "rebaseWhen": "conflicted",
-  "packageRules": [
-    {
-      "matchDatasources": ["orb"],
-      "matchUpdateTypes": ["patch", "minor"],
-      "automerge": true,
-      "automergeType": "branch",
-      "semanticCommitType": "ci"
-    },
-    {
-      "matchDepTypes": ["provided", "test", "build", "import", "parent"],
-      "matchUpdateTypes": ["patch", "minor"],
-      "automerge": true,
-      "automergeType": "branch",
-      "semanticCommitType": "chore"
-    }
-  ]
-}
\ No newline at end of file
+    "extends": ["github>gravitee-io/renovate-config:policy"]
```

```suggestion
    "extends": ["github>gravitee-io/renovate-config:plugin"]
```
policy preset is deprecated: https://github.com/gravitee-io/renovate-config#available-presets

---

done